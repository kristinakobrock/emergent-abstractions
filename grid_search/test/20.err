  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 26.85it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 23.36it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 26.39it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 25.87it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 23.13it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 24.38it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 23.65it/s] 50%|##############################################                                              | 25/50 [00:01<00:01, 23.24it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 22.97it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 23.70it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 25.26it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 25.37it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 28.01it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 28.47it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 27.30it/s]100%|############################################################################################| 50/50 [00:01<00:00, 26.45it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.37it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 21.95it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.56it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 20.18it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.89it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.03it/s]
