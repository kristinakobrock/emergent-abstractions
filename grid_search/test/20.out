# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7119930386543274, "acc": 0.5768756866455078, "length": 3.4868414402008057, "mode": "train", "epoch": 1}
{"loss": 0.6658366918563843, "acc": 0.6249999403953552, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6697783470153809, "acc": 0.6197072267532349, "length": 3.259082555770874, "mode": "train", "epoch": 2}
{"loss": 0.6894195079803467, "acc": 0.5449219346046448, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.662631094455719, "acc": 0.6263065934181213, "length": 3.195101737976074, "mode": "train", "epoch": 3}
{"loss": 0.6638082265853882, "acc": 0.6277344226837158, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6613940596580505, "acc": 0.6310653686523438, "length": 3.16247820854187, "mode": "train", "epoch": 4}
{"loss": 0.6748948097229004, "acc": 0.6109375357627869, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6605421304702759, "acc": 0.6279711723327637, "length": 3.12656569480896, "mode": "train", "epoch": 5}
{"loss": 0.6692332029342651, "acc": 0.624804675579071, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6591300368309021, "acc": 0.6310905814170837, "length": 3.1546576023101807, "mode": "train", "epoch": 6}
{"loss": 0.6642733812332153, "acc": 0.6273437142372131, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6574486494064331, "acc": 0.6311964392662048, "length": 3.180058717727661, "mode": "train", "epoch": 7}
{"loss": 0.6680225133895874, "acc": 0.6273437142372131, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6572049260139465, "acc": 0.6310896277427673, "length": 3.215561628341675, "mode": "train", "epoch": 8}
{"loss": 0.6608538627624512, "acc": 0.6279327869415283, "length": 1.1328125, "mode": "test", "epoch": 8}
{"loss": 0.656146228313446, "acc": 0.6310476064682007, "length": 3.2177083492279053, "mode": "train", "epoch": 9}
{"loss": 0.6610646843910217, "acc": 0.6275696158409119, "length": 3.40234375, "mode": "test", "epoch": 9}
{"loss": 0.6533670425415039, "acc": 0.6314640641212463, "length": 3.150353193283081, "mode": "train", "epoch": 10}
{"loss": 0.6523077487945557, "acc": 0.6290512681007385, "length": 3.65625, "mode": "test", "epoch": 10}
{"loss": 0.6464044451713562, "acc": 0.6315672397613525, "length": 3.301025390625, "mode": "train", "epoch": 11}
{"loss": 0.6411792039871216, "acc": 0.6359894275665283, "length": 3.94921875, "mode": "test", "epoch": 11}
{"loss": 0.6335994601249695, "acc": 0.6393489837646484, "length": 3.5955958366394043, "mode": "train", "epoch": 12}
{"loss": 0.6264125108718872, "acc": 0.6519530415534973, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.613922119140625, "acc": 0.6692847013473511, "length": 3.8368144035339355, "mode": "train", "epoch": 13}
{"loss": 0.6129139065742493, "acc": 0.6634765267372131, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5897455215454102, "acc": 0.6917012333869934, "length": 3.9148356914520264, "mode": "train", "epoch": 14}
{"loss": 0.6092118620872498, "acc": 0.6666015982627869, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5692012906074524, "acc": 0.701506495475769, "length": 3.9645633697509766, "mode": "train", "epoch": 15}
{"loss": 0.6132009625434875, "acc": 0.6691405773162842, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5573040246963501, "acc": 0.713808536529541, "length": 3.972391128540039, "mode": "train", "epoch": 16}
{"loss": 0.6052650809288025, "acc": 0.6796875, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5475722551345825, "acc": 0.7216720581054688, "length": 3.972386360168457, "mode": "train", "epoch": 17}
{"loss": 0.5979990363121033, "acc": 0.6798828840255737, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5421661734580994, "acc": 0.725898027420044, "length": 3.9799094200134277, "mode": "train", "epoch": 18}
{"loss": 0.5973470211029053, "acc": 0.6875000596046448, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5336927771568298, "acc": 0.730678915977478, "length": 3.9854445457458496, "mode": "train", "epoch": 19}
{"loss": 0.5924899578094482, "acc": 0.6949219107627869, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5249915719032288, "acc": 0.7378480434417725, "length": 3.9854025840759277, "mode": "train", "epoch": 20}
{"loss": 0.5863214731216431, "acc": 0.7029296159744263, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5203595161437988, "acc": 0.7418839335441589, "length": 3.987870454788208, "mode": "train", "epoch": 21}
{"loss": 0.5738896727561951, "acc": 0.7017578482627869, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5093377828598022, "acc": 0.7525236010551453, "length": 3.991654396057129, "mode": "train", "epoch": 22}
{"loss": 0.5746104717254639, "acc": 0.7107422351837158, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.5009971857070923, "acc": 0.7568562030792236, "length": 3.99631404876709, "mode": "train", "epoch": 23}
{"loss": 0.5518480539321899, "acc": 0.7242187857627869, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4926724135875702, "acc": 0.764859676361084, "length": 3.995954751968384, "mode": "train", "epoch": 24}
{"loss": 0.5442876815795898, "acc": 0.7265625, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4833492636680603, "acc": 0.7702143788337708, "length": 3.997530221939087, "mode": "train", "epoch": 25}
{"loss": 0.5395344495773315, "acc": 0.7257812023162842, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.478354275226593, "acc": 0.7748613953590393, "length": 3.9935343265533447, "mode": "train", "epoch": 26}
{"loss": 0.5384207963943481, "acc": 0.7355470061302185, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.4740564227104187, "acc": 0.7761914730072021, "length": 3.998100519180298, "mode": "train", "epoch": 27}
{"loss": 0.5249013304710388, "acc": 0.7416015267372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.4637430012226105, "acc": 0.7830957770347595, "length": 3.998450756072998, "mode": "train", "epoch": 28}
{"loss": 0.5121283531188965, "acc": 0.7458984851837158, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.45544251799583435, "acc": 0.7886095643043518, "length": 3.9982657432556152, "mode": "train", "epoch": 29}
{"loss": 0.5088254809379578, "acc": 0.7490234375, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.44779613614082336, "acc": 0.7904716730117798, "length": 3.9987380504608154, "mode": "train", "epoch": 30}
{"loss": 0.5030648708343506, "acc": 0.7541016936302185, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.4446313977241516, "acc": 0.793026328086853, "length": 3.9991109371185303, "mode": "train", "epoch": 31}
{"loss": 0.5025671124458313, "acc": 0.7503905892372131, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.4388306736946106, "acc": 0.7974820733070374, "length": 3.998412847518921, "mode": "train", "epoch": 32}
{"loss": 0.5098134279251099, "acc": 0.7511718273162842, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.43487316370010376, "acc": 0.7978436350822449, "length": 3.9986870288848877, "mode": "train", "epoch": 33}
{"loss": 0.5065388083457947, "acc": 0.7513671517372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.428604394197464, "acc": 0.7985381484031677, "length": 3.9990386962890625, "mode": "train", "epoch": 34}
{"loss": 0.514483630657196, "acc": 0.7580077648162842, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.4224676489830017, "acc": 0.8052228093147278, "length": 3.997753143310547, "mode": "train", "epoch": 35}
{"loss": 0.5045263767242432, "acc": 0.7572265267372131, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.4205109775066376, "acc": 0.8023151159286499, "length": 3.9996094703674316, "mode": "train", "epoch": 36}
{"loss": 0.4996546506881714, "acc": 0.7603515982627869, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.4159686267375946, "acc": 0.80683833360672, "length": 3.999608278274536, "mode": "train", "epoch": 37}
{"loss": 0.501186728477478, "acc": 0.7652344107627869, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.4117198586463928, "acc": 0.8105922341346741, "length": 3.9981963634490967, "mode": "train", "epoch": 38}
{"loss": 0.4888974726200104, "acc": 0.7671875357627869, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.40898412466049194, "acc": 0.8120613098144531, "length": 3.9993953704833984, "mode": "train", "epoch": 39}
{"loss": 0.4873579144477844, "acc": 0.7707030773162842, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.40276676416397095, "acc": 0.8155182003974915, "length": 3.9963057041168213, "mode": "train", "epoch": 40}
{"loss": 0.4862450361251831, "acc": 0.7699218392372131, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.39848366379737854, "acc": 0.817542552947998, "length": 3.997081756591797, "mode": "train", "epoch": 41}
{"loss": 0.48626279830932617, "acc": 0.7710937857627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.39279553294181824, "acc": 0.8207309246063232, "length": 3.996492624282837, "mode": "train", "epoch": 42}
{"loss": 0.4710472524166107, "acc": 0.7820311188697815, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3930094242095947, "acc": 0.8221400380134583, "length": 3.999030828475952, "mode": "train", "epoch": 43}
{"loss": 0.47991251945495605, "acc": 0.7851563096046448, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.38848012685775757, "acc": 0.8248135447502136, "length": 3.9994921684265137, "mode": "train", "epoch": 44}
{"loss": 0.4819958508014679, "acc": 0.7814452648162842, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.38658156991004944, "acc": 0.8260403275489807, "length": 3.9998362064361572, "mode": "train", "epoch": 45}
{"loss": 0.4741882383823395, "acc": 0.7837890982627869, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3776361644268036, "acc": 0.8278847336769104, "length": 3.9976112842559814, "mode": "train", "epoch": 46}
{"loss": 0.48380380868911743, "acc": 0.7765624523162842, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3764888346195221, "acc": 0.8305471539497375, "length": 3.998967170715332, "mode": "train", "epoch": 47}
{"loss": 0.4809277057647705, "acc": 0.7812499403953552, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3727913200855255, "acc": 0.8340300917625427, "length": 3.9994122982025146, "mode": "train", "epoch": 48}
{"loss": 0.4717937707901001, "acc": 0.7878906726837158, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.36061835289001465, "acc": 0.8412905335426331, "length": 3.999692916870117, "mode": "train", "epoch": 49}
{"loss": 0.4608275890350342, "acc": 0.7919922471046448, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3602205216884613, "acc": 0.8448910713195801, "length": 3.9996232986450195, "mode": "train", "epoch": 50}
{"loss": 0.4734557867050171, "acc": 0.794921875, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.35606351494789124, "acc": 0.8469116687774658, "length": 3.999405860900879, "mode": "train", "epoch": 51}
{"loss": 0.44970813393592834, "acc": 0.7999999523162842, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.346653014421463, "acc": 0.8522580862045288, "length": 3.9998745918273926, "mode": "train", "epoch": 52}
{"loss": 0.461489200592041, "acc": 0.8011718392372131, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3448351323604584, "acc": 0.8519830107688904, "length": 3.998863458633423, "mode": "train", "epoch": 53}
{"loss": 0.4518146514892578, "acc": 0.8025389909744263, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3426458239555359, "acc": 0.8572445511817932, "length": 3.9998393058776855, "mode": "train", "epoch": 54}
{"loss": 0.4556552767753601, "acc": 0.8033203482627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.33772972226142883, "acc": 0.8578065037727356, "length": 3.999525785446167, "mode": "train", "epoch": 55}
{"loss": 0.4297002851963043, "acc": 0.8121094107627869, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.33196160197257996, "acc": 0.8577702641487122, "length": 3.9987194538116455, "mode": "train", "epoch": 56}
{"loss": 0.43814587593078613, "acc": 0.8085938096046448, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.33147117495536804, "acc": 0.8600268959999084, "length": 3.9997780323028564, "mode": "train", "epoch": 57}
{"loss": 0.4287404716014862, "acc": 0.8154297471046448, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.322573184967041, "acc": 0.8664079308509827, "length": 3.9997801780700684, "mode": "train", "epoch": 58}
{"loss": 0.4251716136932373, "acc": 0.8162108659744263, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.31761959195137024, "acc": 0.869073212146759, "length": 3.9996964931488037, "mode": "train", "epoch": 59}
{"loss": 0.44621986150741577, "acc": 0.8152343034744263, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.31480568647384644, "acc": 0.8694287538528442, "length": 3.9999475479125977, "mode": "train", "epoch": 60}
{"loss": 0.43264567852020264, "acc": 0.8197265863418579, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7015516757965088
